extensions/amp-story/0.1/page-advancement.js

```diff
@@ -295,14 +296,29 @@ class ManualAdvancement extends AdvancementConfig {
     }, /* opt_stopAt */ this.element_);
   }
 
+  /**
+   * We want clicks on certain elements to be exempted from normal page navigation
+   * @param {!Event} event
+   * @return {boolean}
+   */
+  isProtectedTarget_(event) {
+    const protectedElements = map({
```

Comment:
Constants like this should be hoisted outside of functions to avoid repeated allocations.